app/resources/forms.js.template

```diff
@@ -376,13 +376,13 @@ function make_form_expandable(expandedSection) {
 }
 
 // Show the side menu
-function menuOpenNav() {
-  document.getElementById("mySidenav").style.width = "250px";
-  document.getElementById("overLay").style.width = "100%";
+function showSideMenu() {
+  document.getElementById("overlay").style.display = "block";
+  document.getElementById("sidenav").style.maxWidth = "250px";
 }
 
 // Close the side menu
-function menuCloseNav() {
-  document.getElementById("mySidenav").style.width = "0";
-  document.getElementById("overLay").style.width = "0";
-}
\ No newline at end of file
+function closeSideMenu() {
```

Comment:
Nitpick, but it's better to use paired names for this kind of paired functions. Here you should use either "show/hide" or "open/close".